include/libtorrent/kademlia/node.hpp

```diff
@@ -253,6 +254,8 @@ class TORRENT_EXTRA_EXPORT node
 
 	static protocol_descriptor const& map_protocol_to_descriptor(udp protocol);
 
+	socket_manager* m_sock_man;
```

Comment:
did this move just to order it among other pointers?

Reply:
I moved it up to keep it next to `m_sock` because they usually get accessed together.